### DISCUSSION TITLE
Cant flash simple firmware to drone

### DISCUSSION THREAD

**Discussion by Kiesmith** [bitcraze/discussions#610] [Q&A]
I have been trying to flash my firmware to my drone which I have not done in a while as I have been programming my AI deck. 

I decided to retry the "getting started with development" and changing the LED on the guide. 

I follow each part and everything is fine and builds well until the last part. 

**When I enter "make cload", I get this error:** 

tools/kbuild/Makefile.kbuild:147: warning: overriding recipe for target 'cload'
Makefile:142: warning: ignoring old recipe for target 'cload'
make[1]: Entering directory '/home/bitcraze/projects/crazyflie-firmware/build'
python3 -m cfloader  flash  cf2.bin stm32-fw
/home/bitcraze/miniconda3/bin/python3: No module named cfloader.__main__; 'cfloader' is a package and cannot be directly executed
make[1]: *** [/home/bitcraze/projects/crazyflie-firmware/Makefile:142: cload] Error 1
make[1]: Leaving directory '/home/bitcraze/projects/crazyflie-firmware/build'
make: *** [tools/kbuild/Makefile.kbuild:150: sub-make] Error 2

**Also, I usually get some weird errors everytime I start a new VM and update all projects. Heres a picture of what I receive:** 
![image](https://user-images.githubusercontent.com/72498028/223497344-e69f5e53-7e6b-4d78-8fa4-287252315a65.png)

![image](https://user-images.githubusercontent.com/72498028/223497269-3956cdce-bc6f-4a99-8606-c6e51efbf784.png)

I have not been able to fix it. 
I have had this happen on multiple VMs. It only really worked on my first or second VM but they do not exist anymore.

**Comment by hmllr** [comment]
Hi :)
I think #545 is the same (and #588) - pip is not up to date in the VM anymore which can cause issues. So try to run `pip3 install --upgrade pip`
Best,
Hanna
